content/c/concepts/linked-list/linked-list.md

```diff
@@ -0,0 +1,43 @@
+---
+Title: 'Linked List'
+Description: 'Linked list is a type of data structure in which elements are not sequentially stored but are randomly spread in memory.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Data Structures'
+  - 'Linked Lists'
+  - 'Pointers'
+  - 'Memory'
+  - 'Variables'
+CatalogContent:
+  - 'learn-c'
+  - 'paths/computer-science'
+---
+
+A **`linked list`** is a type of linear data structure in which elements are not sequentially stored but are randomly spread in memory consisting of small units called `node`. These units connect and point to other nodes to create a linked list. A node is the smallest unit of the linked list, responsible for storing two different types of values first is the value we are storing, which can be of any data type, and the second is a pointer used for storing the address of the next node.
```

Comment:
```suggestion
A **linked list** is a type of linear data structure in which elements are not stored sequentially but are dynamically allocated in memory, consisting of small units called nodes. These units connect and point to other nodes to create a linked list.
```

Reply:
Removed the last line, because you've already discussed it in the below structure of a node.